airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/ssh/SshBastionContainer.java

```diff
@@ -78,6 +78,10 @@ public void stopAndCloseContainers(final JdbcDatabaseContainer<?> db) {
     db.close();
   }
 
+  public void stop() {
```

Comment:
nit: to be consistent with `stopAndCloseContainers` I'd either rename this to `stopAndClose` or create another version of `stopAndCloseContainers` with no arguments.